frontend/src/metabase/admin/databases/components/DatabaseList/DatabaseList.jsx

```diff
@@ -75,7 +76,7 @@ export default class DatabaseList extends Component {
 
     return (
       <div className={CS.wrapper} data-testid="database-list">
-        <section className="PageHeader px2 clearfix">
+        <section className={cx("clearfix", AdminS.PageHeader, CS.px2)}>
```

Comment:
I'd advice leaving the order as is so it's very easy to compare to the previous PRs.

```suggestion
        <section className={cx(AdminS.PageHeader, CS.px2, "clearfix")}>
```